ui/src/app/modules/settings/settings.component.html

```diff
@@ -64,6 +64,80 @@ <h5 class="primary-text">
         <i class="fas fa-fw fa-arrow-right"></i>
       </a>
     </li>
+    <li class="list-group-item d-block d-md-flex justify-content-between align-items-center">
```

Comment:
we could only show these fields if the terminal is enabled in general. here is the logic for showing (or not) the terminal icon in the sidebar
- https://github.com/homebridge/homebridge-config-ui-x/blob/1b3df40391a4bd5d2eefc91727a1ccf2cfe78c73/ui/src/app/shared/layout/sidebar/sidebar.component.html#L121

we don't need to worry about `isAdmin` because only an admin can get to the settings page anyway